Hardcore.lua

```diff
@@ -754,6 +749,35 @@ function Hardcore:RequestTimePlayed()
 	RequestTimePlayed()
 end
 
+function Hardcore:ShouldShowPlaytimeWarning(level, percentage)
+	if level <= 5 then
+		return false
+	elseif level <= 15 then
+		return percentage <= 40
+	elseif level <= 20 then
+		return percentage <= 70
+	elseif level <= 25 then
+		return percentage <= 80
+	elseif level <= 30 then
+		return percentage <= 85
+	elseif level <= 35 then
+		return percentage <= 90
+	else
+		return percentage <= 93
+	end
+end
+
+function Hardcore:DisplayPlaytimeWarning(level)
+	local messageprefix = "\124cffFF0000"
+
+	if level <= 20 then
+		Hardcore:Print(messageprefix.."Detected that the player's addon active time is much lower than played time. If you have just installed the addon, start a new character.")
+	else
+		Hardcore:Print(messageprefix.."Detected that the player's addon active time is much lower than played time. If you have just installed the addon: consider starting a new character. Continuing on means you risk your lv 60, HC Verified Status.")
```

Comment:
At this point, would we really invalidate someones run if the percentage is too large. I imagine some people just dc a lot